### ISSUE TITLE
How to genarate *.bin in  FeatureDir

### ISSUE THREAD

**Issue by Hjy20255** [TheLegendAli/DeepLab-Context2#16]
@TheLegendAli 
in the  SetupENV.m
is_server       = 1;

crf_load_mat    = 0;   % the densecrf code load MAT files directly (no call SaveMatAsBin.m)
                       % used ONLY by DownSampleFeature.m
learn_crf       = 0;   % NOT USED. Set to 0

is_mat          = 0;   % the results to be evaluated are saved as mat (1) or png (0)
has_postprocess = 1;   % has done densecrf post processing (1) or not (0)
is_argmax       = 0;   % the output has been taken argmax already (e.g., coco dataset). 
                       % assume the argmax takes C-convention (i.e., start from 0)

debug           = 0;   % if debug, show some results
in  the  run.py
RUN_TRAIN = 0  # Training #1 (on train_aug)
RUN_TEST = 0  # Test #1 specification (on val or test)
RUN_TRAIN2 = 0  # Training #2 (finetune on trainval_aug)
RUN_TEST2 = 0  # Test #2 on official test set
RUN_SAVE = 0  # Translate and save the model
RUN_DENSECRF = 1  # To Run Densecrf
GRID_SEARCH =0  # To Run ONLY if you dont know what parameters to use for Densecrf

i  use  my  own dataset (3 classes)  with your  deeplabV2 code  ,i train  successfully.
BUT  when i run  crf,  i meet  some  error  which  all   *.png  are  black.  

Percentage of pixels correctly labelled overall: 51.387%
Accuracy for each class (pixel accuracy)
      background: 100.000%
      greenhouse:  0.000%
    plastic_film:  0.000%
-------------------------
Mean Class Accuracy: 33.333%
Accuracy for each class (intersection/union measure)
      background: 51.387%
      greenhouse:  0.000%
    plastic_film:  0.000%
-------------------------
Average accuracy: 17.129%
